src/apis/v1/UserExam/service.ts

```diff
@@ -0,0 +1,60 @@
+import { ErrorCodes, HttpException } from 'exceptions';
+import UserExamModel from 'models/schema/UserExam';
+import { logger } from 'utils/logger';
+
+import { UserExamDto, UpdateUserExamDto } from './dto/UserExamDto';
+
+export const getUserExams = async () => {
+  try {
+    const userExam = await UserExamModel.find().populate('author').populate('exam');
+    logger.info(`Get user exams succesfully`);
+
+    return userExam;
+  } catch (error) {
+    logger.error(`Error while get user exame: ${error}`);
+    throw new HttpException(400, ErrorCodes.BAD_REQUEST.MESSAGE, ErrorCodes.BAD_REQUEST.CODE);
+  }
+};
+
+export const createUserExam = async (input: UserExamDto) => {
+  try {
+    const userExam = await UserExamModel.create(input);
+    logger.info(`Create user exams succesfully`);
+
+    return userExam;
+  } catch (error) {
+    logger.error(`Error while create user exame: ${error}`);
+    throw new HttpException(400, ErrorCodes.BAD_REQUEST.MESSAGE, ErrorCodes.BAD_REQUEST.CODE);
+  }
+};
+
+export const updateUserExam = async (input: UpdateUserExamDto, id: string) => {
+  try {
+    const userExam = await UserExamModel.findOneAndUpdate(
+      { _id: id },
+      {
+        author: input.author,
```

Comment:
use $set to update, and i think don't need update delete field, we will create a specific api for that.